dnf-behave-tests/features/plugins-core/debug-dump.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Test for debug plugin - dumping
+
+
+Scenario: dnf debug-dump dumps file with configuration
+  Given I enable plugin "debug"
+    And I use repository "debug-plugin"
+    And I successfully execute dnf with args "install kernel-4.19.1 kernel-4.20.1"
+   When I execute dnf with args "debug-dump {context.dnf.tempdir}/dump.txt"
+   Then the exit code is 0
+    And file "//{context.dnf.tempdir}/dump.txt" matches line by line
+    """
+    dnf-debug-dump version 1
+    %%%%SYSTEM INFO
+      uname: .*
+      rpm ver: .*
+      python ver: .*
+    %%%%DNF INFO
+      arch: .*
+      basearch: .*
+      releasever: .*
+      dnf ver: .*
+      enabled plugins: debug
+      global excludes: 
+    %%%%RPMDB PROBLEMS
+    %%%%RPMDB
+      kernel-0:4\.19\.1-fc29\.x86_64
+      kernel-0:4\.20\.1-fc29\.x86_64
+    %%%%REPOS
+    %debug-plugin - file:///home/mblaha/work/debug-restore-bz1844533/ci-dnf-stack/dnf-behave-tests/fixtures/repos/debug-plugin
```

Comment:
I believe this is your local path and it doesn't match in general. We could hard code the path from our container `%debug-plugin - file:///opt/behave/fixtures/repos/debug-plugin`, but I think we should use another regex here, since the tests can be run in different setups as well.

Reply:
Oh, sorry. I'll fix it.